kbase-extension/scss/partials/_bulkImportCell.scss

```diff
@@ -0,0 +1,150 @@
+/*
+    bulk import cell
+*/
+
+.kb-bulk-import {
+  &__base-node {
+    // everything lives under this
+  }
```

Comment:
I've left some placeholders in here for now -- they can be removed later when it's clear what is and isn't needed. The bulk import cell, like the app cell that it was modelled on, suffers from divitis, so it's likely that quite a few of these can be removed. 🤞

Reply:
I've started working on treating divitis, this'll be great to merge before going much further.